proto/zetachain/zetacore/observer/crosschain_flags.proto

```diff
@@ -19,6 +19,10 @@ message GasPriceIncreaseFlags {
   // Maximum number of pending crosschain transactions to check for gas price
   // increase
   uint32 maxPendingCctxs = 5;
+
+  // Retry interval for Bitcoin chain pending cctx gas price increase
```

Comment:
Why using a different vallue compared to other chain? block time?

Reply:
It will take 30 minutes for a BTC outbound to finalize.